appveyor.yml

```diff
@@ -115,7 +106,7 @@ after_build:
   - py.test tests\unit
 
   # Python integration tests
-  - py.test tests\integration
+  # - py.test tests\integration
```

Comment:
In general, integration tests are considered part of the primary test suite for CI. I don't care what we do in AV but we should be runnign integration tests for every run in Bamboo.